sanic/app.py

```diff
@@ -875,6 +875,8 @@ async def handle_request(self, request: Request):  # no cov
         :param request: HTTP Request object
         :return: Nothing
         """
+        __tracebackhide__ = True
```

Comment:
What's this?

Reply:
Tracerite feature (also shared by some other backtrace formatters, as well as IPython internals) to hide that frame (tab in tracerite). This is used for a few parts of Sanic internals that would otherwise very often appear, to make tidier and only show user code.

It probably would get hidden anyway when Sanic is installed on a system but we really need to hide this for `pip install -e` as well, even though in that case we want to show other Sanic internals.